__tests__/server.test.js

```diff
@@ -3,70 +3,69 @@
 const supertest = require('supertest');
 const { server } = require('../src/server');
 const mockRequest = supertest(server);
-const { dbInstance } = require('../src/models/index');
-const { DESCRIBE } = require('sequelize/types/query-types');
-const { INET } = require('sequelize');
+const { dbConnection } = require('../src/models/index');
+// const { describe } = require('sequelize/types/query-types');
+//const get?
 
-DESCRIBE('web server', () => {
+describe('web server', () => {
   beforeAll(async () => {
-    await dbInstance.sync();
+    await dbConnection.sync();
   });
   afterAll(async () => {
-    await dbInstance.drop();
+    await dbConnection.drop();
   });
   it('should respond with a 404 on an invalid route', () => {
-    return mockRequest.get('/foobar').then((results) => {
+    return mockRequest.get('/food').then((results) => {
       expect(results.status).toBe(404);
     });
   });
   it('should respond with a 404 on an invalid method', async () => {
-    const response = await mockRequest.put('/foobar');
+    const response = await mockRequest.put('/food');
     expect(response.status).toBe(404);
   });
   it('can create a record', async () => {
-    const person = {
-      firstName: 'MJ',
-      lastName: 'McBarksalot',
+    const newFood = {
+      name: 'apple',
     };
-    const response = await mockRequest.post('/people').send(person);
+    const response = await mockRequest.post('/food').send(newFood);
     expect(response.status).toBe(200);
 
     expect(response.body.id).toBeDefined();
 
-    Object.keys(person).forEach((key) => {
-      expect(person[key]).toEqual(response.body[key]);
+    Object.keys(newFood).forEach((key) => {
+      expect(newFood[key]).toEqual(response.body[key]);
     });
   });
 
   it('can get list of records', async () => {
-    const response = await mockRequest.get('/people');
+    const response = await mockRequest.get('/food');
     expect(response.status).toBe(200);
     expect(Array.isArray(response.body)).toBeTruthy();
     expect(response.body.length).toEqual(1);
   });
 
   it('can get a record', async () => {
-    const response = await mockRequest.get('/people/1');
+    const response = await mockRequest.get('/food/1');
     expect(response.status).toBe(200);
-    expect(typeof response.body).toEqual('object');
+    expect(typeof response.body).toEqual('object/:id');
```

Comment:
'object/:id' is not a [JavaScript type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)